### PR TITLE
Change Junit system test names to include function args

### PR DESCRIPTION
**Pull request by manosnoam**
Signed-off-by: nmanos <nmanos@redhat.com>

### CODE REVIEW

File: setup_subm.sh [hunk 1]
```diff
@@ -5252,9 +5252,10 @@ function debug_test_pass() {
   trap_to_debug_commands;
   PROMPT "PASS test for DEBUG"
 
-  local test="TRUE"
+  local arg1="$1"
```
Comment (MaxBab) [inline setup_subm.sh:5255]: What those arg1 and arg2 are used for?
Why they should be equal?

Comment (manosnoam) [reply setup_subm.sh:5255]: This is just a dummy test I used to check that the junit is indeed created with the input args.
I tested it on the "debug" branch, before adding it to this branch.
This is the job I used to debug it:
https://qe-jenkins-csb-skynet.apps.ocp4.prod.psi.redhat.com/view/Maintenance/job/Maintenance/job/Debug_job/192/

This is the output in Polarion which now includes unique names for the test with args:
https://polarion.stage.engineering.redhat.com/polarion/#/project/RHACM4K/testrun?id=multi-cluster-networking_debug-job-sys&tab=records

Comment (manosnoam) [reply setup_subm.sh:5255]: sure, let me add this info

Comment (MaxBab) [reply setup_subm.sh:5264]: Thanks

**Review by MaxBab — APPROVED** [review]

**Pull request merged**